frontend/src/components/ExchangePage.vue

```diff
@@ -289,7 +289,10 @@ export default {
     getRequests() {
       this.loadingTable = true;
       this.requests = [];
-      ApiService.apiAxios.get(Routes.edx.EXCHANGE_URL, {params: {pageNumber: this.pageNumber - 1, pageSize: this.pageSize}})
+      const sort = {
+        sequenceNumber: 'DESC',
+      };
+      ApiService.apiAxios.get(Routes.edx.EXCHANGE_URL, {params: {pageNumber: this.pageNumber - 1, pageSize: this.pageSize, sort}})
```

Comment:
I think we are supposed to sort by descending create date (I hadn't implemented this). PEN-1795